spec/features/mechanics/show_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "Mechanic Show Page", type: :feature do
+  before :each do
+    @six_flags = AmusementPark.create!(name: 'Six Flags', admission_cost: 75)
+    @hurler = @six_flags.rides.create!(name: 'The Hurler', thrill_rating: 7, open: true)
+    @scrambler = @six_flags.rides.create!(name: 'The Scrambler', thrill_rating: 4, open: true)
+
+    @mechanic = Mechanic.create!(name: 'Kara Smith', years_experience: 11)
+    @mechanic.rides << [@hurler, @scrambler]  # Associate mechanic with rides
+  end
+
+  it "displays the mechanic details and rides they are working on" do
+    visit mechanic_path(@mechanic)
+
+    expect(page).to have_content(@mechanic.name)
+    expect(page).to have_content(@mechanic.years_experience)
+    expect(page).to have_content(@hurler.name)
+    expect(page).to have_content(@scrambler.name)
+  end
```

Comment:
A couple of things to consider for improving this test suite. Add a third ride that is not associated with the mechanic, and assert that this third ride does not show up on the page. To reduce the chance of false positive assertions, make use of `within` blocks or anchor/label text, especially when it comes to finding numbers on a page. These have the highest chance of producing false positives.

```ruby
expect(page).to have_content("Year of Experience: #{@mechanic.years_experience}")
```